.github/workflows/ispc-ci-san.yml

```diff
@@ -0,0 +1,41 @@
+name: Sanitizers
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+env:
+  LLVM_REPO: https://github.com/ispc/ispc.dependencies
+
+jobs:
+  linux-ispc-llvm15-asan:
+    runs-on: ubuntu-latest
+    env:
+      LLVM_VERSION: "15.0"
+      LLVM_TAR: llvm-15.0.7-ubuntu18.04-Release+Asserts-x86.arm.wasm.tar.xz
+      ISPC_OPAQUE_PTR_MODE: "ON"
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Install dependencies
+      run: |
+        .github/workflows/scripts/install-build-deps.sh
+
+    # We will use system gcc until we rebuild LLVM with compiler-rt.
+    - name: Check environment
+      run: |
+        ./check_env.py
+        which -a gcc
+        cat /proc/cpuinfo
+
+    # Address sanitizer includes leak sanitizer but we don't want it to report
+    # all leaks at the moment because there are quite a lot if them.
+    # So, we disable leaks detection to have clean report.
+    - name: Run ASAN
```

Comment:
Let's have 3 different steps here for clarity / readability of the run:
- config and build ISPC with ASAN enabled
- run ASAN on LIT tests
- run ASAN on runtime tests

Reply:
Fixed, test run is here https://github.com/nurmukhametov/ispc/actions/runs/5535409990/jobs/10101667972

Reply:
Example of failed job on double-free error is here https://github.com/nurmukhametov/ispc/actions/runs/5535736851/jobs/10102438055